src/traces/parcoords/plot.js

```diff
@@ -0,0 +1,51 @@
+/**
+* Copyright 2012-2017, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var parcoords = require('./parcoords');
+var Lib = require('../../lib');
+
+module.exports = function plot(gd, cdparcoords) {
+
+    var fullLayout = gd._fullLayout;
+    var svg = fullLayout._paper;
+    var root = fullLayout._paperdiv;
+    var data = cdparcoords.map(function(d, i) {
```

Comment:
On the off chance, this PR comment lost alignment: I see it as linked to `var data = cdparcoords.map(function(d, i) {` in `plots.js`.

This expression takes `cdparcoords` that's passed to the exported `plot` function, and binds it to variable `data` that's passed on as input for the `parcoords` function call on the bottom of `plot.js`. I'm planning to phase out a line from  the function body ('item._gdDimensions = gd.data[i].dimensions;').